fluentd/README.md

```diff
@@ -0,0 +1,9 @@
+# Fluentd
+[Fluentd](https://www.fluentd.org/) is the log collector that resides on each Openshift node to gather application and node logs
+
+## Configuration
+Following are the environment variables that can be modified to adjust the configuration:
+
+| Environment Variable | Description |Example|
+|----------------------|-------------|---|
+| `OCP_OPERATIONS_PROJECTS`| The list of project for which messages will be sent to the operations indices|`OCP_OPERATIONS_PROJECTS="journal.system** system.var.log** **_default_** **_openshift_** **_openshift-** audit.log**"`
```

Comment:
It isn't really a list of projects - it is a list of fluentd tag patterns which match records tagged as operations records.